src/Common/src/System/Globalization/IdnMapping.cs

```diff
@@ -82,18 +82,22 @@ public string GetAscii(string unicode, int index, int count)
             Contract.EndContractBlock();
 
             // We're only using part of the string
```

Comment:
agreed - this is my fault accidently not deleting the substring section - I will remove and add tests